mathesar_ui/src/component-library/collapsible/Collapsible.scss

```diff
@@ -3,25 +3,29 @@
 
   .collapsible-header {
     display: flex;
-    width: 100%;
-    text-align: left;
-    cursor: pointer;
-    font-weight: inherit;
-
-    .collapsible-header-title {
-      flex-grow: 1;
-      overflow: hidden;
-      font-weight: 600;
-    }
-    svg {
-      flex-shrink: 0;
-      flex-grow: 0;
-    }
-    &:active,
-    &.active,
-    &:focus,
-    &.focus {
-      box-shadow: none;
+    justify-content: space-between;
+    .collapsible-header-btn {
+      display: flex;
+      width: 100%;
+      text-align: left;
+      cursor: pointer;
+      font-weight: inherit;
+      align-items: flex-start;
```

Comment:
This particular CSS seems to be causing icon misalignment here: 
<img width="419" alt="Screenshot 2023-01-31 at 12 14 44 AM" src="https://user-images.githubusercontent.com/11032856/215567135-fa453687-44f5-4ccd-8eaa-5e69fd524943.png">

Reply:
Fixed icon misalignment issue
![Screenshot from 2023-01-31 22-35-51](https://user-images.githubusercontent.com/38041443/215832417-c11aa805-f04c-4535-983d-e5a518f83e43.png)